__tests__/index.test.ts

```diff
@@ -13,9 +13,13 @@ expect.extend({
 
 addSerializer({
   test: (val: any) => typeof val === 'string',
-  print: (v: any) => v as string,
+  print: (v: any) => cleanUpVersion(v),
 });
 
+function cleanUpVersion(str: string): string {
+  return str.replace(/"version":\s(\".*\"),\s*/g, '');
```

Comment:
I would keep version to check if it is not removed in the future:

```suggestion
  return str.replace(/"version":\s(\".*\"),\s*/g, '"version": "<version>"');
```